libraries/AP_GPS/GPS_Backend.h

```diff
@@ -79,6 +79,12 @@ class AP_GPS_Backend
         return _last_itow;
     }
 
+    enum DriverOptions : int16_t {
```

Comment:
This is an odd storage type.

Reply:
This was just to reflect the fact that we are using this against a `AP_Int16` everywhere, so just trying to minimize casts/extensions.